plexpy/webserve.py

```diff
@@ -2519,6 +2519,42 @@ def get_plays_by_stream_type(self, time_range='30', y_axis='plays', user_id=None
             logger.warn("Unable to retrieve data for get_plays_by_stream_type.")
             return result
 
+    @cherrypy.expose
+    @cherrypy.tools.json_out()
+    @requireAuth()
+    @addtoapi()
+    def get_concurrent_streams_by_stream_type(self, time_range='30', **kwargs):
+        """ Get graph data for concurrent streams by stream type by date.
+
+            ```
+            Required parameters:
+                None
+
+            Optional parameters:
+                time_range (str):       The number of days of data to return
+
+            Returns:
+                json:
+                    {"categories":
+                        ["YYYY-MM-DD", "YYYY-MM-DD", ...]
+                     "series":
+                        [{"name": "Direct Play", "data": [...]}
+                         {"name": "Direct Stream", "data": [...]},
+                         {"name": "Transcode", "data": [...]}
+                         ]
+                     }
+            ```
+        """
+
+        graph = graphs.Graphs()
+        result = graph.get_total_concurrent_streams_per_stream_type(time_range=time_range)
+        logger.debug(result)
```

Comment:
```suggestion

```

Reply:
**User Filter**
With the change introduced in #2090 I agree that in can be included now although it will need a change in the logic.

**Tooltip**
Instead of the `Total: 4` it would be more useful to display the `max count of concurrent streams on per day` like already done on the start page.